__tests__/query.test.js

```diff
@@ -0,0 +1,133 @@
+const { graphql } = require('graphql');
+const schema = require('../schema/schema');
+
+describe("getAllUsers query: ", () => {
+  it('• should return all user data from test seeds', async () => {
+    const EXPECTED = {
+      "data": {
+        "getAllUsers": [
+          {
+            "id": "1",
```

Comment:
We'll have to make sure if we ever change our seeds we'll have to change this file as well

Reply:
Yes. Thankfully, we'll only have to change it once per file since this constant is used throughout the file as a reference for expected values. :)